webapp/tests/forms/steps/lotse/test_personal_data_steps.py

```diff
@@ -527,15 +527,19 @@ def test_if_number_over_25_chars_then_succ_validation(self, new_test_request_con
 class TestPersonAHasDisabilityValidation:
     def test_if_required_value_is_given_then_validation_should_be_success(self, new_test_request_context):
         data = MultiDict({'person_a_has_disability': 'yes'})
-        with new_test_request_context(form_data=data):
+        with new_test_request_context(form_data=data) as req:
+            req.session = SecureCookieSession(
+                {_LOTSE_DATA_KEY: create_session_form_data(data)})
```

Comment:
Why did you have to add this? Is this because of an earlier step?

Reply:
sry misunderstood that :grin:

Reply:
I think this is still there?